Wrappers/Python/cil/framework/framework.py

```diff
@@ -3276,12 +3495,30 @@ def dtype(self):
     def dtype(self, val):
         self.__dtype = val      
         
+    @property
+    def dimension_labels(self):
+
+        if self.__dimension_labels is None:
```

Comment:
I just made this consistent with the other geometry classes. This might need changing in many places in our code.

With get/set methods I don't think it's an issue as the child classes set the correct variable. But it's probably something that needs changing elsewhere too.